docusaurus.config.js

```diff
@@ -16,7 +16,7 @@ const config = {
   url: 'https://siglens.github.io',
   // Set the /<baseUrl>/ pathname under which your site is served
   // For GitHub pages deployment, it is often '/<projectName>/'
-  baseUrl: '/siglens-docs/',
+  baseUrl: '/',
```

Comment:
I don't think this change should be merged(it can be reverted after Andrew has reviewed the netlify link), it might impact the Github pages deployment.

Reply:
Ohh Yes, I will update it, once Andrew has reviewed.

Reply:
I think Venky is right. But Mani, I don't quite understand; were you waiting for me to review the markdown files, or waiting for me to find if this will mess up the url/deployment?

Reply:
No this is just because, `Netlify` would throw an error if the baseURL is not set to `/`. 
So I believe when Venky made the initial deployment setup he might have set the url to `/` and we used Venky's `spl-docs` branch(which is the same branch deployed on Netlify). 
And I pushed that here without noticing it.

I said I would update once you have reviewed, because if I did now the Netlify link that is in PR description will not work. So I wanted to update it once you have reviewed everything and there are no further changes.

Reply:
I have updated this baseURL now to the original one.